manifests/m/MartinKinkelin/RoboMirror/2.0/MartinKinkelin.RoboMirror.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MartinKinkelin.RoboMirror
+PackageVersion: 2.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://deac-ams.dl.sourceforge.net/project/robomirror/RoboMirror/RoboMirror%202.0%20Setup.exe
```

```suggestion
  InstallerUrl: https://sourceforge.net/projects/robomirror/files/RoboMirror/RoboMirror%202.0%20Setup.exe/download
```
Just in case the mirror goes down and no one is able to download it.